Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive.yml

```diff
@@ -1713,17 +1779,1590 @@ script:
+      type: String
+    - contextPath: GoogleDrive.FilePermission.FilePermission.photoLink
+      description: A link to the user's profile photo, if available.
+      type: String
+  - name: google-drive-file-permission-update
+    description: Updates a permission with patch semantics.
+    deprecated: false
+    arguments:
+    - name: file_id
+      description: id of the requested file.
+      required: false
+      isArray: false
+      defaultValue: ""
+      predefined:
+      - ""
+    - name: user_id
+      description: The user's primary email address.
+      required: false
+      isArray: false
+      defaultValue: ""
+      predefined:
+      - ""
+    - name: expiration_time
+      description: The time at which this permission will expire (RFC 3339 date-time).
+      required: false
+      isArray: false
+      defaultValue: ""
+      predefined:
+      - ""
+    - name: permission_id
+      description: The ID of the permission.
+      required: false
+      isArray: false
+      defaultValue: ""
+      predefined:
+      - ""
+    - auto: PREDEFINED
+      default: false
+      defaultValue: "reader"
+      description: The role granted by this permission. Default is reader.
+      isArray: false
+      name: role
+      predefined:
+      - "owner"
+      - "organizer"
+      - "fileOrganizer"
+      - "writer"
+      - "commenter"
+      - "reader"
+      required: false
+      secret: false
+    outputs:
+    - contextPath: GoogleDrive.FilePermission.FilePermission.deleted
+      description: Whether the account associated with this permission has been deleted.
+        This field only pertains to user and group permissions.
+      type: Boolean
+    - contextPath: GoogleDrive.FilePermission.FilePermission.displayName
+      description: The "pretty" name of the value of the permission.
+      type: String
+    - contextPath: GoogleDrive.FilePermission.FilePermission.emailAddress
+      description: The email address of the user or group to which this permission
+        refers.
+      type: String
+    - contextPath: GoogleDrive.FilePermission.FilePermission.id
+      description: The ID of this permission.
+      type: String
+    - contextPath: GoogleDrive.FilePermission.FilePermission.kind
+      description: Identifies what kind of resource this is.
+      type: String
+    - contextPath: GoogleDrive.FilePermission.FilePermission.role
+      description: The role granted by this permission.
+      type: String
+    - contextPath: GoogleDrive.FilePermission.FilePermission.type
+      description: The type of the grantee.
+      type: String
+    - contextPath: GoogleDrive.FilePermission.FilePermission.photoLink
+      description: A link to the user's profile photo, if available.
+      type: String
+  - name: google-drive-file-permission-delete
+    description: Delete a permission.
+    deprecated: false
+    arguments:
+    - name: file_id
+      description: id of the requested file.
```

Comment:
for any argument of this kind, specify how to find it (which command returns it).

Reply:
https://github.com/demisto/content/commit/b6ab42c24c2aa6bf80b6e626853f01917eb0aead